packages/gestalt/src/HelpButton.js

```diff
@@ -0,0 +1,185 @@
+// @flow strict
+import { type Node, type Element, useRef, useState, useEffect, useId } from 'react';
+import Box from './Box.js';
+import TapArea, { type OnTapType } from './TapArea.js';
+import Popover from './Popover.js';
+import Text from './Text.js';
+import Link from './Link.js';
+import Icon from './Icon.js';
+import Flex from './Flex.js';
+import Tooltip from './Tooltip.js';
+import { type Indexable, CompositeZIndex, FixedZIndex } from './zIndex.js';
+import { useDefaultLabelContext } from './contexts/DefaultLabelProvider.js';
+
+type Props = {|
+  /**
+   * Supply a short, descriptive label for screen-readers to describe the popover content. Used for [accessibility](https://gestalt.pinterest.systems/web/popover#ARIA-attributes) purposes.
+   */
+  accessibilityPopoverLabel: string,
+  /**
+   * Specifies the preferred position of the tooltip and popover relative to HelpButton. See [Popover's ideal direction variant](https://gestalt.pinterest.systems/web/popover#Ideal-direction) to learn more.
+   */
+  idealDirection?: 'up' | 'right' | 'down' | 'left',
+  /**
+   * If provided, displays a link at the bottom of the popover message.
```

Comment:
I would link here to the Link page and mentioned to follow guidance from Link component.

Reply:
Done